@xen-orchestra/web/src/pages/pools.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <VtsNoDataHero v-if="!servers.length" type="page" />
+  <div v-else class="pools">
+    <UiCard v-if="isReady" class="pools-table">
```

Comment:
```suggestion
    <UiCard class="pools-table">
```
You already have `isReady` from `useServerStore` in `PoolsTable`